### ISSUE TITLE
Trident Get fatal error: concurrent map writes after upgrading to 20.07.01

### ISSUE THREAD

**Issue by khankth** [NetApp/trident#456]
**Describe the bug**
After upgrading to 20.07.01, it is noted that the trident orchestrator pod restarts once every day on average with below messages, looking into trident-main container logs following error is seen:
```
fatal error: concurrent map writes

goroutine 88934 [running]:
runtime.throw(0x1f23efe, 0x15)
        /usr/local/go/src/runtime/panic.go:1116 +0x72 fp=0xc001c7d3f8 sp=0xc001c7d3c8 pc=0x435752
runtime.mapdelete_faststr(0x1c5d020, 0xc000c108d0, 0xc001dfa9c0, 0x28)
        /usr/local/go/src/runtime/map_faststr.go:306 +0x37f fp=0xc001c7d460 sp=0xc001c7d3f8 pc=0x4151ff
github.com/netapp/trident/frontend/csi.(*Plugin).CreateVolume(0xc000ba60e0, 0x229c7c0, 0xc001b2ce70, 0xc0027622a0, 0xc0022a1cb0, 0x0, 0x0)
        /go/src/github.com/netapp/trident/frontend/csi/controller_server.go:213 +0xe06 fp=0xc001c7da20 sp=0xc001c7d460 pc=0x19ef5a6
...
```
Attached is the full error logs:
[20.07.01-logs.txt](https://github.com/NetApp/trident/files/5327919/20.07.01-logs.txt)
**Environment**
- Trident version: [20.07.01]
- Trident installation flags used: [tctl install --k8s-timeout 30m --csi --use-custom-yaml --silence-autosupport]
- Container runtime: [ Docker 19.03.13-CE]
- Kubernetes version: [1.19.2]
- Kubernetes orchestrator: [k8s kubeadm]
- OS: [ RHEL 7.8]
- NetApp backend types: [ONTAP(main), ontap-san]

**Additional context**
We upgraded recently from 20.07.00 to 20.07.01 after upgrading kubernetes from 1.18.4 to 1.19.2

**Comment by gnarl** [comment]
Hi @khankth,

We've looked at the logs you've provided and reviewed the code involved. We are unable to determine why concurrent map writes are occurring in your case.

Can you please contact NetApp Support and provide additional log files?

To open a case with NetApp, please go to https://mysupport.netapp.com/site/.

- Bottom left, Click on 'Contact Support'
- Find the appropriate number from your region to call in, or login.
- Note: Trident is not listed on the page, but is a supported product by NetApp based on a supported Netapp storage SN.
- Open the case on the NetApp storage SN, and provide the description of the problem.
- Be sure to mention the product is Trident on Kubernetes, and provide the details. Mention this GitHub.

The case will be directed to Trident support engineers for response.

**Comment by gnarl** [comment]
Hi @khankth,

One of our engineers continued to look at this issue after the initial assessment. A fix is included with commit [0a245d3](https://github.com/NetApp/trident/commit/0a245d3895af31f910a58c2f26e5a0f8b25f34f8). This fix will be included in the Trident 20.10 release.

**Comment by khankth** [comment]
Thanks @gnarl  for the update!

**Comment by gnarl** [comment]
This fix is included in Trident v20.10.0.
